### PR TITLE
Add additional steps to Dual boot instructions in prerequisites.md

### PR DESCRIPTION
**Pull request by Mogashni**
- [x] You have read [The Odin Projects Contributing Guide](https://github.com/TheOdinProject/curriculum/blob/main/CONTRIBUTING.md).
 - [x] The PR title summarizes the change and where it happened, for example: "Fixes punctuation in Clean Code lesson".
 
#### 1.Describe the changes made and include why they are necessary or important:

Added an additional step 3.3 on how to access and setup boot properties in BIOS/UEFI so that the computer boots with Unbuntu

Reason: Once Ubuntu is installed if you don't set up your pc to boot with it in BIOS settings then it will boot from windows or you will have to do a long workaround which is time consuming.

#### 2. If this PR is related to an open issue please reference it with the `#` operator and the issue number below:

#XXXXX

### CODE REVIEW

**Comment by Mogashni** [comment]
Windows/Ubuntu Dual boot - Step 3.3 added the below text

#### Step 3.3: Configure BIOS/UEFI settings

Now you are going to set up your boot settings so that when you boot your pc it will open up Ubuntu straight away and also give you the option to boot with Windows. 

First you have to locate your BIOS/UEFI settings on your pc.
For step-by-step instructions, please follow this [instruction guide](https://www.laptopmag.com/articles/access-bios-windows-10)

Second you will need to locate the boot settings in your BIOS/UEFI.
For step-by-step instructions, please follow this [instruction guide](https://www.lifewire.com/change-the-boot-order-in-bios-2624528)

In the boot settings you are looking to change it to boot from Ubuntu. If you don't have this option you can also select USB.

File: foundations/installations/prerequisites.md [hunk 1]
```diff
@@ -188,6 +188,18 @@ Installing Ubuntu is where the real changes start happening on your computer. Th
 
 For step-by-step instructions, please follow this [installation guide](https://tutorials.ubuntu.com/tutorial/tutorial-install-ubuntu-desktop#0) from the creators of Ubuntu.
 
+#### Step 3.3: Configure BIOS/UEFI settings
+
+Now you are going to set up your boot settings so that when you boot your pc it will open up Ubuntu straight away and also give you the option to boot with Windows. 
+
```
Comment (kashura) [inline foundations/installations/prerequisites.md:194]: This needs to include a sentence about the fact that the exact steps will vary between computers.

**Review by kashura — CHANGES_REQUESTED** [review]

**Comment by kashura** [comment]
Thank you for taking initiative on submitting this. Let me bring this to the attention of the team.

**Comment by kashura** [comment]
Thank you for making a contribution. We are going to close this PR at this time.

**Pull request closed without merging**